fixtures/blizzard/lil-g.json

```diff
@@ -0,0 +1,400 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Lil' G",
+  "categories": ["Moving Head", "Color Changer"],
+  "meta": {
+    "authors": ["Ryan Goodwin"],
+    "createDate": "2023-09-18",
+    "lastModifyDate": "2023-09-18"
+  },
+  "links": {
+    "manual": [
+      "https://cdn.shopify.com/s/files/1/1741/5055/files/lil-g_manual_rev_a.pdf"
+    ],
+    "productPage": [
+      "https://www.blizzardpro.com/products/lil-g"
+    ],
+    "video": [
+      "https://www.youtube.com/watch?v=-lwu9weWlBk"
+    ],
+    "other": [
+      "https://cdn.shopify.com/s/files/1/1741/5055/files/lil-g_cutsheet.pdf"
+    ]
+  },
+  "physical": {
+    "dimensions": [201, 317, 196],
+    "weight": 3.5,
+    "power": 59,
+    "powerConnectors": {
+      "powerCON": "input only"
```

Comment:
The picture in the manual shows IEC, not powercon.

Reply:
The company I work for owns 2 of these units are they are both powercon in similar to this one.
<img alt="lilg rear with powercon" src="https://github.com/OpenLightingProject/open-fixture-library/assets/34293941/48cbb533-0247-44df-abaf-1e06f779ea52">

I honestly missed that the manual had an IEC version in it! I can add that connector type.

Reply:
IEC C13 connector added

Reply:
It's kind of an odd feature to have in OFL, anyway, since many/most fixtures can be found with several types of power connector, or are easily retrofitted.  I'm not sure anyone's even using that data.